### ISSUE TITLE
Font issue on the documentation website.

### ISSUE THREAD

**Issue by dino8890** [wagtail/wagtail#9747]
There is some kind of font/text issue on the Wagtail docs, see screenshot below:

![Screenshot from 2022-12-05 10-36-52](https://user-images.githubusercontent.com/17911579/205603900-9275288b-d898-4237-944c-e1471d30ba25.png)

Whitespace character ` ` is unusually wide, however HTML does not show anything out of the ordinary:

`<p>Multi-site is a Wagtail project configuration where c...`

I

**Comment by lb-** [comment]
Hey. Could you provide any more details?

What browser/ OS ?

What exact URL (just in case you are somehow on a different version of the docs)?

Have you tried incognito/private mode (might be a plugin)

I am not able to reproduce this issue (was ok the docs this morning).

**Comment by dino8890** [comment]
Hello, sorry it's still morning here and I haven't had my coffee yet hah.

The issue appears o _Firefox 107_ running on _Solus Linux 4.3_ (incognito or not). _Google Chrome 101.0.4951.64_ does not have this issue.

Only documentation for versions _latest_, _stable_, _4.1.1_, _4.0.4_, and _4.0.3_ are affected. _3.0.3_ is not affected, nor are the earlier versions.

Happens on this page, as well as any other:
https://docs.wagtail.org/en/stable/advanced_topics/multi_site_multi_instance_multi_tenancy.html

**Comment by lb-** [comment]
Ok - maybe it's a default font issue. 

Could you try to grab a screenshot with the styles inspector open in Firefox. Just to see what font it's actually using. 

You may need to take the screenshot of the 'computed' font family also.

**Comment by dino8890** [comment]
Indeed, it seems like a font availability issue. Here's the tabs for version 4:

**Fonts:**
![fonts](https://user-images.githubusercontent.com/17911579/205622157-f7e730ac-d832-4182-97fb-32fe78d5cc02.png)

**Computed**
![computed](https://user-images.githubusercontent.com/17911579/205622209-96b0959b-bc19-4122-88b0-1ed6526d0124.png)

And then for version 3:

**Fonts**
![fonts_3 0 3](https://user-images.githubusercontent.com/17911579/205622323-e8da647d-239e-441d-b511-d8c061d3b1cb.png)

**Computed**
![computed_3 0 3](https://user-images.githubusercontent.com/17911579/205622372-39bd83c0-d416-430b-8de8-47cef082511a.png)

**Comment by lb-** [comment]
Ok. 

I'm going to move this to the Sphinx theme repo - looks like this broke via https://github.com/wagtail/sphinx_wagtail_theme/pull/194

We will need to maybe find another font to add to the font set. 

Just curious @dino8890 - does the Wagtail admin look ok? I think it uses the same font family set.

**Comment by lb-** [comment]
Wagtail admin UI fonts for reference - https://github.com/wagtail/wagtail/blob/main/client/src/tokens/typography.js

**Comment by dino8890** [comment]
Wagtail admin is okay, it falls back to _Noto Sans_. The blank space issue is caused by rendering whitespace using _Noto Color Emoji_ font for whatever reason.
